README.md

```diff
@@ -111,11 +111,14 @@ In cases where the current working directory makes a difference, the tests
 assume that the working directory is the root of the test suite, so that the
 relative path to itself is `t/spec/S\d\d-$section/$filename`.
 
-You can also run the `fudge` program manually. This command will fudge and run
-`prove` on a specific file:
-
-    ./fudge --keep-exit-code rakudo S05-metasyntax/angle-brackets.t &&
-    prove -eperl6 S05-metasyntax/angle-brackets.rakudo
+You can also run the `fudge` program manually. To fudge and run `prove` on a
+specific file:
```

Comment:
It says `run the fudge program manually` but the command runs `fudgeandrun`. The text probably needs to be updated?